### DISCUSSION TITLE
Simple Contact Form 2 setup/help

### DISCUSSION THREAD

**Discussion by tvadovic** [typemill/typemill#488] [Q&A]
Hi,
i'm trying to add contact form on a page, i did setup the plugin, inserted the path to page where to include the form on the webpage, but still nothing happens. is there some shortcode or anything else that needs to be added/done on the specific page where i want the form to show?

also, i'm not exactly programmer, although i know some code and with a tutorial i can manage some things. is there a tutorial for dummies, where can i learn how to edit this plugin to show other fields and make it a new public form? 

i'm trying to migrate a small school site from wordpress to typemill, and public forms are one of the requirements. other option is to embed google forms, but as of know from what i've read here i think that's not supported in typemill.

thanks!

t.

**Comment by trendschau** [comment]
There is just the path and if it is correct the form should show up in frontend (only in frontend). Do you get any error messages? You can enable error messages in the developer settings with a checkbox (display error details). 

For a forms plugin, you can use the contact form plugin, follow the general plugin tutorial on the website for a basic understanding (https://typemill.net/plugin-developers/tutorial), and read the chapter about public forms and forms in general: https://typemill.net/forms/public-forms

Good look with your project, would be nice to hear your experience with migration from WordPress

Best regards
Sebastian

**Comment by tvadovic** [comment]
No error messages, it just doesn't show up on frontend.
This is the page where i want it to show: https://vadovic.eu/typemill/o-nas
In the plugin, I set up this path: **/o-nas** (screenshot attached)

![plugin_settings](https://github.com/user-attachments/assets/ad885fb4-6641-4e12-b730-437ff5c06e5a)

**Comment by trendschau** [comment]
did you try with /typemill/o-nas or typemill/o-nas ?

**Comment by tvadovic** [comment]
i did try a lot of things:
- /typemill/o-nas
- /typemill/o-nas/
- https://vadovic.eu/typemill/o-nas

i even tried an empty page <https://vadovic.eu/typemill/publicform>, in
case that's how it works

i tried other plugin, search, to be sure there is not more general problem
with all the plugins, but search is working

would it help to give you admin access to the backend? this is anyway just
a testing site to evaluate the possibility to migrate from wp

thanks

t.

On Mon, Aug 26, 2024 at 2:16 PM trendschau ***@***.***> wrote:

> did you try with /typemill/o-nas or typemill/o-nas ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/typemill/typemill/discussions/488#discussioncomment-10451167>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BHNDIKNZO3AIN4OIUYH5ZX3ZTMMBVAVCNFSM6AAAAABNCP74YWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANBVGEYTMNY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Tomáš Vadovič

***@***.***
***@***.***
http://www.radiohlavy.sk

**Comment by tvadovic** [comment]
i tried fresh install, and suddenly it's [working ](https://vadovic.eu/slnecnica/napiste-nam/kontakt)without any problem.
